client/overview/task-list/test/index.js

```diff
@@ -44,16 +57,86 @@ describe( 'TaskList', () => {
 		CollapsibleList.mockImplementation( renderChildren );
 		TaskItem.mockImplementation( renderChildren );
 		Text.mockImplementation( renderNothing );
+		const createNotice = jest.fn();
+		useDispatch.mockReturnValue( {
+			createNotice,
+		} );
 	} );
+	it( 'shows an incomplete task', () => {
+		const overviewTasksVisibility = getOverviewTasksVisibilityMock();
+		render(
+			<TaskList
+				tasks={ [ tasksMocked ] }
+				overviewTasksVisibility={ overviewTasksVisibility }
+			/>
+		);
 
-	it( "map task's `onClick` prop to `action`", () => {
-		const task = createTaskMock();
+		expect( TaskItem ).toHaveBeenCalledWith(
+			expect.objectContaining( {
+				title: 'Task Title',
+				completed: false,
+			} ),
+			expect.anything()
+		);
+	} );
+	it( 'does not show deleted tasks', () => {
+		const overviewTasksVisibility = getOverviewTasksVisibilityMock();
+		overviewTasksVisibility.deletedTodoTasks.push( 'task-key' );
+		render(
+			<TaskList
+				tasks={ [ tasksMocked ] }
+				overviewTasksVisibility={ overviewTasksVisibility }
+			/>
+		);
+		expect( screen.queryByText( /Task Title 1/ ) ).not.toBeInTheDocument();
```

Comment:
After changing the mock in the comment above this should be `/Task Title/` instead of `Task Title 1`. Same goes for all the other instances.

Reply:
Good eyes, I added this change in the commit a240722 as well.